devtools/dump_devinfo.py

```diff
@@ -35,6 +35,13 @@ def scrub(res):
         "longitude_i",
         "latitude",
         "longitude",
+        "owner",
+        "device_id",
+        "ip",
+        "ssid",
+        "hw_id",
+        "fw_id",
+        "oem_id",
```

Comment:
This looks good to me for now, but we will probably want to move the list of keys to scrub (including the ones below) into the library itself. This will allow removing the hardcoded list from homeassistant integration: https://github.com/home-assistant/core/blob/b36ddaa15c5975fbae36e2b4eef9721ef45f1a35/homeassistant/components/tplink/diagnostics.py#L14